src/main/java/seedu/address/model/tag/TagsContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.tag;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.person.Person;
+
+/**
+ * Tests that a {@code Person}'s {@code Tags} matches any of the keywords given.
+ */
+public class TagsContainsKeywordsPredicate implements Predicate<Person> {
```

Comment:
Maybe `TagsContainsKeywordsPredicate` and `SkillsContainsKeywordsPredicate` could be refactored into a single class for future iterations?